daml/Marketplace/Issuance/Service.daml

```diff
@@ -69,7 +49,7 @@ template Service
           create ReduceIssuanceRequest with ..
 
     controller provider can
-      nonconsuming Originate: ContractId AssetDescription
+      nonconsuming Originate: (ContractId AssetDescription, AssetDescription)
```

Comment:
This feels wrong ...

Reply:
It avoids having to fetch / query after having executed the choice if you need to work with the result, so I think it could be a valid pattern (`fetchByKey` also returns the same). But we can discuss

Reply:
If you're doing this to have the neat list of originations in `Demo.daml`, note that you can achieve something similar by createive use of a list and `<*>`. Something like:

```
[usd, eur] <- sequence $ curry originate <*> [
  ("USD", "United States Dollar", Zero),
  ("EUR", "Euro", Zero)
]
```

Reply:
> It avoids having to fetch / query after having executed the choice if you need to work with the result, so I think it could be a valid pattern (`fetchByKey` also returns the same). But we can discuss

This is what I thought. The downside is that now you can't do the opposite (just the creation), which can be a problem. I'm thinking of a case where you're creating a batch of instruments, say an options chain. This will add the cost of fetching the trees for each option, even if you throw them away afterward. It's especially costly because of the trees, which need to be serialized/deserialized.